installing/installing_oci/installing-oci-assisted-installer.adoc

```diff
@@ -17,17 +17,12 @@ include::modules/installing-oci-about-assisted-installer.adoc[leveloffset=+1]
 * link:https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform/[{ai-full} for {product-title}]
 * xref:../../installing/installing_platform_agnostic/installing-platform-agnostic.adoc#cluster-entitlements_installing-platform-agnostic[Internet access for {product-title}]
 * link:https://docs.oracle.com/en-us/iaas/Content/Block/Concepts/blockvolumeperformance.htm#vpus[Volume Performance Units (Oracle documentation)]
+* link:https://docs.oracle.com/iaas/Content/openshift-on-oci/installing-agent-about-instance-configurations.htm[Instance Sizing Recommendations for {product-title} on {oci} Nodes]
```

Comment:
```suggestion
* link:https://docs.oracle.com/iaas/Content/openshift-on-oci/installing-agent-about-instance-configurations.html[Instance Sizing Recommendations for {product-title} on {oci} Nodes]
```
I assume this needs to be `html` rather than `htm`. However, I am getting a 500 error even with the `html`.

Reply:
Thanks. The Oracle docs should be live later today so I can validate the URLs tomorrow.